atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -219,10 +219,17 @@ def run(self):
             top_layer, _ = pulp_push_results
             annotations['v1-image-id'] = top_layer
 
+        media_types = []
```

Comment:
If you make it a set to start with here, you don't need to create a set from it later.

Reply:
`set` cannot concatenate lists (coming from `pulp_pull_results`). I could turn them all into sets, but it feels a bit like cheating

Reply:
Fair enough.